sci-astronomy/wcstools/wcstools-3.9.5.recipe

```diff
@@ -0,0 +1,227 @@
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+defineDebugInfoPackage wcstools$secondaryArchSuffix \
+	"$commandBinDir"/addpix$commandSuffix \
+	"$commandBinDir"/bincat$commandSuffix \
+	"$commandBinDir"/char2sp$commandSuffix \
+	"$commandBinDir"/conpix$commandSuffix \
+	"$commandBinDir"/cphead$commandSuffix \
+	"$commandBinDir"/delhead$commandSuffix \
+	"$commandBinDir"/delwcs$commandSuffix \
+	"$commandBinDir"/edhead$commandSuffix \
+	"$commandBinDir"/filename$commandSuffix \
+	"$commandBinDir"/fileroot$commandSuffix \
+	"$commandBinDir"/filext$commandSuffix \
+	"$commandBinDir"/fixpix$commandSuffix \
+	"$commandBinDir"/getcol$commandSuffix \
+	"$commandBinDir"/getfits$commandSuffix \
+	"$commandBinDir"/gethead$commandSuffix \
+	"$commandBinDir"/getpix$commandSuffix \
+	"$commandBinDir"/gettab$commandSuffix \
+	"$commandBinDir"/i2f$commandSuffix \
+	"$commandBinDir"/imcat$commandSuffix \
+	"$commandBinDir"/imextract$commandSuffix \
+	"$commandBinDir"/imfill$commandSuffix \
+	"$commandBinDir"/imhead$commandSuffix \
+	"$commandBinDir"/immatch$commandSuffix \
+	"$commandBinDir"/imresize$commandSuffix \
+	"$commandBinDir"/imrot$commandSuffix \
+	"$commandBinDir"/imsize$commandSuffix \
+	"$commandBinDir"/imsmooth$commandSuffix \
+	"$commandBinDir"/imstack$commandSuffix \
+	"$commandBinDir"/imstar$commandSuffix \
+	"$commandBinDir"/imwcs$commandSuffix \
+	"$commandBinDir"/isfile$commandSuffix \
+	"$commandBinDir"/isfits$commandSuffix \
+	"$commandBinDir"/isnum$commandSuffix \
+	"$commandBinDir"/isrange$commandSuffix \
+	"$commandBinDir"/keyhead$commandSuffix \
+	"$commandBinDir"/newfits$commandSuffix \
+	"$commandBinDir"/scat$commandSuffix \
+	"$commandBinDir"/sethead$commandSuffix \
+	"$commandBinDir"/setpix$commandSuffix \
+	"$commandBinDir"/simpos$commandSuffix \
+	"$commandBinDir"/sky2xy$commandSuffix \
+	"$commandBinDir"/skycoor$commandSuffix \
+	"$commandBinDir"/sp2char$commandSuffix \
+	"$commandBinDir"/subpix$commandSuffix \
+	"$commandBinDir"/sumpix$commandSuffix \
+	"$commandBinDir"/wcscrlf$commandSuffix \
+	"$commandBinDir"/wcsgetdate$commandSuffix \
+	"$commandBinDir"/wcshead$commandSuffix \
+	"$commandBinDir"/wcsremap$commandSuffix \
+	"$commandBinDir"/xy2sky$commandSuffix \
+	"$libDir"/libwcstools.so.$libVersion
+
+PATCH()
+{
+	sed -i \
+		-e 's/getdate/wcsgetdate/' \
+		-e 's/crlf/wcscrlf/' \
+		-e 's/remap/wcsremap/' \
+		-e "s/3.... Programs/${portVersion} Programs/" \
+		wcstools
+}
+
+BUILD()
+{
+	for f in "$portDir"/additional-files/{configure.ac,wcstools.pc.in,Makefile.am}; do
+		cp ${f} .
+	done
+	cp "$portDir"/additional-files/Makefile.libwcs.am libwcs/Makefile.am
+
+	mkdir -p m4
+
+	libtoolize --force --copy --install
+	aclocal --force --install
+	autoheader --force
+	autoconf --force
+	automake --add-missing --force-missing
```

Comment:
Can't you combine these as "autoreconf -vfi"?

Reply:
No, because -i in autoreconf won't call the libtoolize with --install (or -i) flagmand I need it for a couple of m4 macros (a but in the upstream package, most probably).

I could call libtoolize AND autoreconf, but this will end calling libtoolize two times (autoreconf will still call libtoolize by its own), which is ugly.

I think it's better to let them split in the various, single, commands in these circumstances. :)